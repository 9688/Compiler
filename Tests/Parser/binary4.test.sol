"ROOT"
    "EXPR"
         "+"
              "a"
              "*"
                   "b"
                   "c"
    "EXPR"
         "+"
              "*"
                   "a"
                   "b"
              "c"
    "EXPR"
         "+"
              "+"
                   "a"
                   "*"
                        "b"
                        "c"
              "d"
    "EXPR"
         "+"
              "*"
                   "a"
                   "b"
              "*"
                   "c"
                   "d"
    "EXPR"
         "*"
              "a"
              "EXPR"
                   "+"
                        "b"
                        "c"
    "EXPR"
         "*"
              "EXPR"
                   "+"
                        "a"
                        "b"
              "c"
