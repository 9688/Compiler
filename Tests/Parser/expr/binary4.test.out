
list
     "+"
          a
          "*"
               b
               c
list
     "+"
          "*"
               a
               b
          c
list
     "+"
          "+"
               a
               "*"
                    b
                    c
          d
list
     "+"
          "*"
               a
               b
          "*"
               c
               d
list
     "*"
          a
          list
               "+"
                    b
                    c
list
     "*"
          list
               "+"
                    a
                    b
          c

